client/src/components/GameBoxScore.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+import { useMediaQuerySSR } from 'hooks'
+import { VStack, HStack, Table, Thead, Tbody, Tr, Th, Td, Box } from '@chakra-ui/react'
+
+const Totals = ({ playersStats }) => {
```

Comment:
Just wondering for this, since this is the total points of the playerStats, wouldn't this be the equivalent to the team's totalPoints? If thats the case, you might be able to just pass that in without doing the calculation here again

Reply:
Although thinking about it now, your way is technically cleaner since in the future we may want stats other than totalPoints, so I'm gonna resolve this comment